build.sbt

```diff
@@ -323,7 +325,7 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
         }
       }
     }.value,
-  dockerBaseImage := "bitnami/java:11-prod",
+  dockerBaseImage := "openjdk:8-jre-slim-buster",
```

Comment:
Can you extract `"openjdk:8-jre-slim-buster"` into a val, and then write a short comment describing why this is the base image?